### PR TITLE
added Software Developers IG to community page

### PR DESCRIPTION
**Pull request by aeschylus**

### CODE REVIEW

**Comment by aeschylus** [comment]
This updates the community groups page with the software developers interest group description and pointers to it.

**Comment by zimeon** [comment]
It seems odd to include the sentence "This interest group includes software developers from Digirati, Harvard University, Princeton University, and Stanford University." -- that seems potentially off-putting to others?

**Comment by jpstroop** [comment]
+1, I don't see much value in that sentence. "Developers from active
members of the IIIF community" or something like that.

**Comment by mikeapp** [comment]
Agree with @zimeon and @jpstroop

File: source/community/groups/software/index.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: "Software Developers Interest Group"
+layout: spec
+tags: [components, software development, implementation]
+cssversion: 2
+---
+The Software Developers Interest Group formed in response to the growing need for shared, composable libraries for IIIF viewer software following the IIIF Meeting in New York in May 2016. Software developers from several institutions involved in developing user experiences on top of the IIIF APIs were duplicating each other’s effort and finding it difficult to consistently apply the specifications.
```
Comment (azaroth42) [inline source/community/groups/software/index.md]: each other's efforts

File: source/community/groups/software/index.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+As the IIIF developer community grows, it continues to have a greater need for reusable components that can be composed into special-purpose applications. In the interest of keeping pace with improvement to the Image and Presentation API specifications, as well as the introduction of new APIs, the interest group provides a vehicle for developers from multiple institutions to discuss, develop, and curate a collection of software components that adhere to certain standards of software quality, composability, and specification compliance.  This interest group includes software developers from Digirati, Harvard University, Princeton University, and Stanford University.
```
Comment (azaroth42) [inline source/community/groups/software/index.md]: :+1: to removing the list of some of the institutions, per other comments

**Comment by aeschylus** [comment]
Okay, I can fix this up and remove the institution names (and typo). We based our description on the Manuscripts IG description, which lists institutions.

**Comment by azaroth42** [comment]
We should have Manuscripts remove their list, or include it at the bottom and keep it up to date.

**Comment by aeschylus** [comment]
@jpstroop Struck that sentence, left "effort" in the singular for religious reasons.

File: source/community/groups/software/index.md (hunk 1 above)
Comment (jpstroop) [inline source/community/groups/software/index.md]: @aeschylus also :arrow_up:

**Comment by aeschylus** [comment]
@jpstroop Changed effort to efforts for religious reasons.

**Comment by aeschylus** [comment]
👍

**Comment by aeschylus** [comment]
I agree it sounds awkward, my hairsplitting explanation was that each place had one effort, and so "each" "other"'s effort was interfering with "each" "other"'s effort.

**Comment by jpstroop** [comment]
:+1:

**Comment by zimeon** [comment]
belated :+1:

**Pull request merged**